checkstyle/suppressions.xml

```diff
@@ -195,12 +195,7 @@
     <suppress checks="JavadocMethod" files="com/hazelcast/partition/"/>
     <suppress checks="JavadocType" files="com/hazelcast/partition/"/>
     <suppress checks="JavadocVariable" files="com/hazelcast/partition/"/>
-    <suppress checks="MethodCount|ExecutableStatementCount|FileLengthCheck|MagicNumber"
-              files="com/hazelcast/internal/partition/impl/InternalPartitionServiceImpl"/>
-    <suppress checks="ClassFanOutComplexity|ClassDataAbstractionCoupling"
-              files="com/hazelcast/internal/partition/impl/InternalPartitionServiceImpl"/>
-    <suppress checks="CyclomaticComplexity|NPathComplexity"
-              files="com/hazelcast/internal/partition/impl/PartitionStateGeneratorImpl"/>
+    <suppress checks="FileLengthCheck" files="com/hazelcast/internal/partition/impl/InternalPartitionServiceImpl"/>
```

Comment:
For unknown reasons this check doesn't work with `@SuppressWarnings`, so it has to remain here :(